static/src/javascripts/projects/commercial/modules/third-party-tags/audience-science-pql.js

```diff
@@ -28,36 +28,52 @@ define([
     }
 
     function onLoad() {
-        var asiPlacements = window.asiPlacements;
-        var segments = storage.local.get(storageKey) || {};
-        // override the global value with our previously stored one
-        window.asiPlacements = segments[section];
-        segments[section] = asiPlacements;
-        storage.local.set(storageKey, segments);
+        isLoaded = true;
+        mediator.emit('commercial:audience-science:loaded');
```

Comment:
In the end I removed the `shouldLazyLoad` test entirely. AS also loads in single-call mode; the test was initially used to _not_ add the callback that removes keys, but I'm OK to let that go.